src/Main.hs

```diff
@@ -33,24 +38,21 @@ main = execParser opts >>= run
 run :: Config -> IO ()
 run (Config w p) = do
     c <- contentsOrSTDIN p
-    putStrLn $ rstrip . unlines $ wrapLine w =<< lines c
+    T.putStrLn $ T.stripEnd . T.unlines $ wrapLine w =<< T.lines c
```

Comment:
The mixture of `.` and `$` here is sort of weird -- why not replace the `.` with `$`? Or replace all the `$` with `.`.